src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.impl.FileReaderServiceImpl;
+import core.basesyntax.impl.FileWriterServiceImpl;
+import core.basesyntax.impl.FruitStoreServiceImpl;
+import core.basesyntax.impl.ReportServiceImpl;
+import core.basesyntax.impl.ValidatorServiceImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.strategy.ManipulationService;
+import core.basesyntax.strategy.Manipulations;
+import core.basesyntax.strategy.impl.BalanceManipulationService;
+import core.basesyntax.strategy.impl.PurchaseManipulationService;
+import core.basesyntax.strategy.impl.ReturnManipulationService;
+import core.basesyntax.strategy.impl.SupplyManipulationService;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static final String INPUT_PATH = "src/main/resources/InputFile";
+    private static final String OUTPUT_PATH = "src/main/resources/OutputFile";
+
+    public static void main(String[] args) {
+        Map<Manipulations, ManipulationService> manipulationServiceMap = new HashMap<>();

```

Comment:
```suggestion
        Map<Manipulation, ManipulationService> manipulationServiceMap = new HashMap<>();
```
enum name should better be singular